easybuild/easyblocks/g/gromacs.py

```diff
@@ -182,6 +182,12 @@ def sanity_check_step(self):
         if self.toolchain.options.get('usempi', None):
             suff = '_mpi'
 
+        # Add the _d suffix to the suffix, in case of the double precission
+        if '-DGMX_DOUBLE=on' in self.cfg['configopts']:
+            suff = suff + '_d'
+        elif '-DGMX_DOUBLE=ON' in self.cfg['configopts']:
+            suff = suff + '_d'
```

Comment:
@besserox I think this is a good option...

@paderijk up for giving this a shot? I would run `cmake -LA .`, and then use Python regular expressions to check for `GMX_DOUBLE:BOOL=ON` in the output rather than using `grep`...

Quick & dirty try (untested):

```python
out, _ = run_cmd("cmake -LA .", simple=False)
if re.search("GMX_DOUBLE:BOOL=ON", out):
    suff = suff + '_d'
```